code/style.css

```diff
@@ -1,4 +1,341 @@
+  100% { transform: translate(1px, -2px) rotate(-1deg); }
+}
+
+hr {
+  border-bottom: black, solid, 0.5px;
+  margin-left: 25px;
+  margin-right: 25px;
+}
+
+/*** BIG NEWS ***/
+
+.big-news-container {
+  display: flex;
+  flex-direction: column;
+  margin: 180px 20px 0 20px;  
+}
+
+.big-news-container h2 {
+  font-size: 30px;
+}
+
+.big-news-container p {
+  font-size: 18px;
+  line-height: 1.4;
+}
+
+img.top {
+  width: 100%;
+  object-fit: cover;
+}
+
+/*** SMALL NEWS ***/
+
+.small-news-container{
+  margin: 20px;
+}
+
+.cards {
+  flex-wrap: wrap;
+  text-align: center;
+  padding: 20px; 
+  margin-bottom: 20px;
+  box-shadow: 0 2px 5px 0 #d3d3d3;
+  transition: 0.3s;
+}
+
+p {
+    margin-top: -18px;
+}
+
+.cards img {
+  height: auto;
+  width: 100%;
+  overflow: hidden;
+}
+
+footer {
+  background: beige;
+  height: 50px;
+  display: flex;
+  align-items: center;
+  padding: 10px 30px;
+  font-size: 18px;
+  font-weight: lighter;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+footer img {
+  width: 100px;
+}
+
+footer p {
+  margin: 0;
+}
+
+/*** MEDIA QUERIES ***/
+
+@media (min-width: 688px) {
+  
+  header {
+    position: fixed;
+    width: 100%;
+  }
+
+  nav {
+    display: inline-block;
+    order: 0;
+    padding-right: 20px; 
+  }
+
+  .site-name {
+    order: 1;
+    margin-left: auto;
+    margin-right: 20px;
+  }
+
+  header img {
+    order: 2;
+    padding-right: 20px;
+  }
+  
+  .hamburger {
+    display: none;
+  }
+
+  .big-news-container {
+    display: inline-flex;
+    flex-direction: row;
+    padding: 20px 0 30px 0;
+    border-bottom: 2px solid black;
+    margin: 160px 20px 0 20px;
+  }
+
+  img.top {
+    width: 58%;
+    margin-left: 20px;
+    align-self: center;
+  }
+
+  .top-text {
+    margin: 0 25px 0 30px;
+    text-align: left; 
+    align-self: center;
+    max-width: 300px;
+  }
+
+  .small-news-container {
+    display: flex;
+    flex-wrap: wrap; 
+    justify-content: space-evenly;
+  }
+
+  .cards {
+    width: 40%;
+    text-align: left;  
+  }
+
+  hr {
+    display: none;
+  }
+}
+
+@media (min-width: 992px) {
+  
+  header {
+    position: fixed;
+    top: 0;
+    overflow: hidden;
+  }
+
+  nav {
+    display: inline-block;
+    order: 2;
+    margin-left: auto;
+    padding-right: 30px;
+    
+  }
+
+  header img{
+    order: 0;
+  }
+
+  .site-name {
+    order: 1;
+    margin-left: 10px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  main {
+    margin: 0 auto;
+    width: 80%;
+  }
+
+  .big-news-container {
+    display: inline-flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: 150px 20px 0 20px;
+    padding-top: 30px;
+    padding-bottom: 30px;
+  }
+
+  .big-news-container div {
+    text-align: left;
+    width: 38%;
+  }
+
+  img.top {
+    width: 58%;
+    max-height: 600px;
+  }
+
+  .small-news-container {
+    display: flex;
+    flex-wrap: wrap; 
+    justify-content: space-evenly;
+  }
+
+  .cards:hover {
+    opacity: 0.7;
+  }
```

Comment:
Careful with opacity! It makes the text harder to read. Maybe it's enough with some opacity on the image?
In that case, you can write like this:
```
.cards:hover img {
  opacity: 0.7;
}
```
meaning, when you hover the card - apply this styling to the child element `img`